infrastructure/template.yaml

```diff
@@ -250,7 +250,7 @@ Resources:
   PublicKbvHmrcApiAccessLogGroup:
     Type: AWS::Logs::LogGroup
     Properties:
-      LogGroupName: !Sub /aws/vendedlogs/apigateway/${AWS::StackName}-${PublicKbvHmrcApi}-public-AccessLogs
+      LogGroupName: !Sub /aws/vendedlogs/apigateway/${AWS::StackName}-public-AccessLogs
```

Comment:
How come we are changing these names?

Reply:
Hi @gds-jbentham this was raised by the script as a issue, @Z-Mohamoud has detailed the issue in the comments on [LIME-1258](https://govukverify.atlassian.net/browse/LIME-1258)

Reply:
@smsgds could you have a look at this

Reply:
We had a look at this, there is no circular dependancies due to the use of getAttr and Ref. 

- The api is always created first
- Then the loggroup, log group sub filters and api stage are all created next.

So the names can be reverted back

Reply:
Approving as is, the aws-cdk-lib used is unaware of the api gateway being created separately from the stage. Leading to it thinking the gateway name would be unavailable.

The same naming change will probably be needed when template checks are added to fraud, dl and passport.